io/pci/PowerVMEEH.py

```diff
@@ -85,6 +86,8 @@ def test_eeh_basic_pe(self):
             self.pci_mem_addr = pci.get_memory_address(self.addr)
             self.pci_mask = pci.get_mask(self.addr)
             self.pci_class_name = pci.get_pci_class_name(self.addr)
+            if self.pci_class_name == 'fc_host':
```

Comment:
@Naresh-ibm You need to change https://github.com/avocado-framework/avocado/blob/master/avocado/utils/pci.py#L149

Reply:
@narasimhan-v  I already went through this method. but this is specific only for EEH test case, i,e we need scsi_host. we may still need the fc_host as class name for other test. hence changed the test script.

let the library function remain same.

Reply:
@Naresh-ibm okay, if you are fine with it.